package.json

```diff
@@ -110,53 +111,50 @@
     "@types/i18next": "8.4.4",
     "@types/jest": "24.0.15",
     "@types/jest-plugin-context": "2.9.0",
-    "@types/node": "14.0.27",
+    "@types/node": "^16.11.26",
     "@types/react": "16.9.35",
     "@types/react-dom": "16.9.8",
     "@types/react-jsonschema-form": "1.0.10",
     "@types/react-router-dom": "5.1.5",
     "@types/request": "2.48.1",
     "@types/semver": "6.0.2",
     "@types/uuid": "3.4.5",
-    "@types/webpack": "4.41.27",
-    "@types/webpack-dev-server": "3.11.3",
-    "@types/webpack-merge": "4.1.5",
-    "awesome-typescript-loader": "5.2.1",
     "concurrently": "5.3.0",
-    "copy-webpack-plugin": "6.1.0",
-    "css-loader": "1.0.0",
-    "electron": "11.5.0",
-    "electron-builder": "22.8.0",
+    "copy-webpack-plugin": "^6.4.0",
+    "css-loader": "^6.7.0",
+    "css-minimizer-webpack-plugin": "^3.4.1",
+    "electron": "^18.0.0",
+    "electron-builder": "^22.14.13",
```

Comment:
we've generally been locking our version without hat^or ~ thinking we want to drop these hats consistency